npm/private/utils.bzl

```diff
@@ -134,11 +134,16 @@ def _parse_package_name(package):
         return (segments[0], segments[1])
     return ("", segments[0])
 
+def _npm_registry_url(package, registries, default_registry):
+    (package_scope, _) = _parse_package_name(package)
+
+    return registries[package_scope] if package_scope in registries else default_registry
+
 def _npm_registry_download_url(package, version, registries, default_registry):
     "Make a registry download URL for a given package and version"
 
-    (package_scope, package_name_no_scope) = _parse_package_name(package)
-    registry = "https://{}".format(registries[package_scope]) if package_scope in registries else default_registry
+    (_, package_name_no_scope) = _parse_package_name(package)
```

Comment:
looks like this like can go; package_name_no_scope not used either

Reply:
It's a param in the `.format` below

Reply:
ooohhh. i see. missed that. 👍